ui/app/components/app/modals/account-modal-container/account-modal-container.component.js

```diff
@@ -0,0 +1,51 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import Identicon from '../../../ui/identicon'
+
+export default function AccountModalContainer (props, context) {
+  const {
+    selectedIdentity,
+    showBackButton,
+    backButtonAction,
+    hideModal,
+    children,
+  } = props
+
+  return (
+    <div style={{ borderRadius: '4px' }}>
+      <div className="account-modal-container">
+        <div>
+          <Identicon
+            address={selectedIdentity.address}
+            diameter={64}
+          />
+        </div>
+        {showBackButton && (
+          <div className="account-modal-back" onClick={backButtonAction}>
+            <i className="fa fa-angle-left fa-lg" />
+            <span className="account-modal-back__text">{' ' + context.t('back')}</span>
+          </div>
+        )}
+        <div className="account-modal-close" onClick={hideModal} />
+        {children}
+      </div>
+    </div>
+  )
+}
+
+AccountModalContainer.contextTypes = {
+  t: PropTypes.func,
+}
+
+AccountModalContainer.defaultProps = {
+  showBackButton: false,
+  children: false,
```

Comment:
```suggestion
  children: null,
```
I suspect that `false` won't render anyway, but... still not the most sensible default.

Reply:
Good catch, thanks!